os/drivers/lcd/mipi_lcd.c

```diff
@@ -117,6 +117,17 @@ struct mipi_lcd_dev_s {
 
 static struct mipi_lcd_dev_s g_lcdcdev;
 
+static int send_cmd_retry(struct mipi_lcd_dev_s *priv, struct mipi_dsi_msg* msg)
```

Comment:
This function is not in the conditional like
```
#ifdef CONFIG_LCD_...RETRY
static int send_cmd_retry(....)
{
}
#endif
```
So, you should add some more code like
```
#if !defined(CONFIG_LCD_.._RETRY)
#define CONFIG_LCD....RETRY 1
#endif
```

Reply:
BUT `CONFIG....RETRY 1 ` can be understandable that we will retry once.
Undefined the config means that we will not retry.
When you make modification, you should check this as well.